auth/service.go

```diff
@@ -104,17 +102,19 @@ type service struct {
 	groups       GroupRepository
 	idProvider   mainflux.IDProvider
 	ulidProvider mainflux.IDProvider
+	keto         PolicyAgent
```

Comment:
Should this one be called `keto`? On one hand - it is readable. On the other hand, Keto is just a policy provider, or authorization system, and can be potentially changed in the future in Mainflux.

Maybe `policyAgent` here would be a better naming?

@dusanb94 @mteodor @nmarcetic opinions please

Reply:
I vote for `policyAgent`.